src/state/pools/helpers.ts

```diff
@@ -156,13 +157,13 @@ export const transformVault = (vaultKey: VaultKey, vault: SerializedCakeVault):
 
 export const getTokenPricesFromFarm = (farms: SerializedFarm[]) => {
   return farms.reduce((prices, farm) => {
-    const quoteTokenAddress = farm.quoteToken.address.toLowerCase()
-    const tokenAddress = farm.token.address.toLowerCase()
+    const quoteTokenAddress = isAddress(farm.quoteToken.address)
```

Comment:
hmm. `isAddress` function returning `string | boolean` is quite counter-intuitive. Could we replace it with `getAddress` from `@ethersproject/address` instead.

Reply:
I remember getAddress will throw error